### DISCUSSION TITLE
千问的role有哪几种？

### DISCUSSION THREAD

**Discussion by 346407440** [QwenLM/Qwen#670] [Q&A]
除了user以及assistant的role，还有跟gpt一样的system的role吗？
除此之外，千问的训练数据集的格式应该是怎样的？有例子吗？

**Comment by shenxiangzhuang** [comment]
是有`system` role的，可以从[灵积文档](https://help.aliyun.com/zh/dashscope/developer-reference/api-details?disableWebsiteRedirect=true#b9ad0a10cfhpe)看到，目前应该有`system`、`user`、`assistant`三个role

**Comment by 346407440** [comment]
你发对地方了~收到 tang
